lib/rack/session/abstract/id.rb

```diff
@@ -17,6 +17,17 @@ module Abstract
       # SessionHash is responsible to lazily load the session from store.
 
       class SessionHash
+        using Module.new {
+          refine Hash do
+            def transform_keys(&block)
+              hash = {}
+              each do |key, value|
+                hash[block.call(key)] = value
+              end
+              hash
+            end unless {}.respond_to?(:transform_keys)
```

Comment:
Should we move the `{}.respond_to?(:transform_keys)` to the `using` call to avoid refining on Ruby 2.6?

Reply:
You're right. That seems more efficient.